.circleci/config.yml

```diff
@@ -0,0 +1,44 @@
+version: 2
```

Comment:
Thanks for working on automating the publishing Pavel! Can we implement this via github actions though? We purposely switched away from using CircleCI to avoid leaking Transferwise infrastructure details (e.g. artifactory URL).

Reply:
We are checking multiple options how to automate this. Going with GHA is bad because it doesn't limits how credentials are accessible, and they may leak easily. 
Few alternatives are circleci, cloud circleci and some tricky way with 3rd project in the middle - most likely done with Github actions. Currently researching.